typescript/infra/scripts/get-owner-ica.ts

```diff
@@ -27,26 +62,59 @@ function getArgs() {
     .alias('chains', 'destinationChains').argv;
 }
 
+interface IcaDeployResult {
+  chain: string;
+  result?: IcaArtifact;
+  error?: string;
+  deployed?: string;
+  recovered?: string;
+}
+
 async function main() {
   const {
     environment,
     ownerChain,
-    chains,
+    chains: chainsArg,
     deploy,
     owner: ownerOverride,
   } = await getArgs();
   const config = getEnvironmentConfig(environment);
   const multiProvider = await config.getMultiProvider();
 
+  let artifacts: ChainMap<IcaArtifact>;
+  try {
+    artifacts = await readAbacusWorksIcas(environment);
+  } catch (err) {
+    console.error('Error reading artifacts, defaulting to no artifacts:', err);
+    artifacts = {};
```

Comment:
Could this have unintended consequences if someone mistakenly moves the file or renames it?

Reply:
Probably - the reason this doesn't throw is if the file doesn't exist due to valid reasons, e.g. for a new environment. But in the spirit of being extra conservative, happy to throw and leave a note about what to do if it doesn't exist

Reply:
If we are deploying to a new env, we'd need to do a code change right? as in read from a different file maybe?